Dockerfile.dist

```diff
@@ -82,8 +99,10 @@ LABEL \
   com.redhat.component="ec-cli"
 
 # Install cosign and other tools we want to use in the Tekton task
+# Todo: How to prefetch these rpms in the prefetch-dependencies task??
```

Comment:
Watch [this space](https://github.com/containerbuildsystem/cachi2/discussions/465)